remarques/remarques.md

```diff
@@ -0,0 +1,34 @@
+# Page de doc 1 -> https://www.ubik-ingenierie.com/blog/monitor-jmeter-test-from-browser/
+![Legende](images/doc1.png)
+La remarque 3 n'est pas bien illustrée.
+Il faudrait une image ou du texte pour dire que ça vient de add -> listener -> ULP observability
+
+
+# Page de doc 2 -> https://www.ubik-ingenierie.com/blog/ubik-load-pack-observability-plugin/
+![Legende](images/doc2-1.png)
+je n'ai pas l'icone qui est entouré, j'ai du la trouver dans Options -> plugins managers. ça peut être bien de le mentionner si les gens n'ont pas non plus l'icone.
+
+![Legende](images/doc2-2.png)
+Jetty Metrics route -> je comprend pas bien ce qui est affiché sur cette route et à quoi elle sert.
+Je suppose que c'est un truc utile pour le dev du plugin ?
+
+Number of Processing Threads -> En quoi est-ce que c'est utile de faire varier le nombre de threads qui lisent les réponses aux requêtes ? Je suppose que àa optimise les calculs sur les réponses mais c'est pas explicite vu qu'on peut aussi augmenter le nombre de threads ailleurs dans Jmeter et que ça sert à simuler plus de traffique, ce qui n'est pas la même chose.
```

Comment:
Les 2 threads n'ont pas de rapport directs:
- Les threads dans JMeter simulent les Virtual Users (1 Thread == 1 VU)
- Par contre plus ton test sera aggressif soit par le nombre de VUs très élevé soit par la fréquence des requêtes, plus il faut de threads de consommations sinon la queue intermédiaire va être saturée

Reply:
Ok, mais comment on fait pour estimer correctement le nombre de threads à utiliser ? Il faut le piffer un peu et ajuster en fonction des résultats affichés par le plugin ? Si c'est ça peut être qu'il faut un peu plus détailler la doc la dessus.

Reply:
@Liard-Thomas , tu as raison, c'est un axe d'amélioration
Essaie de fournir une formule en fonction de ce que je t'ai expliqué

Reply:
Plus grand mais plus clair :

<b>Number of Processing Threads : </b> Number of threads processing the results of the request. A JMeter thread usually represents a user. For this plugin, a thread is a literal "thread". The more threads you have, the less likely you are to experience slowdowns in the queue containing the data to be consumed. The more users and/or the more frequent requests the test has, the more threads are needed to consume the data in time.